src/main/java/cubicchunks/network/AbstractMessageHandler.java

```diff
@@ -66,9 +66,8 @@ public IMessage onMessage(T message, MessageContext ctx) {
 			// the only reason to check side here is to use our more aptly named handling methods
 			// client side proxy will return the client side EntityPlayer
 			return handleClientMessage(CubicChunks.proxy.getPlayerEntity(ctx), message, ctx);
-		} else {
-			// server side proxy will return the server side EntityPlayer
-			return handleServerMessage(CubicChunks.proxy.getPlayerEntity(ctx), message, ctx);
 		}
+		// server side proxy will return the server side EntityPlayer
+		return handleServerMessage(CubicChunks.proxy.getPlayerEntity(ctx), message, ctx);
```

Comment:
separate cleanup PR would be fine, but mixing that with functional changes makes everything harder